src/Microsoft.Framework.Runtime/Internal/ConcurrencyUtilities.cs

```diff
@@ -19,7 +19,7 @@ internal static string FilePathToLockName(string filePath)
             return filePath.Replace(Path.DirectorySeparatorChar, '_');
         }
 
-        internal async static Task<T> ExecuteWithFileLocked<T>(string filePath, Func<bool, Task<T>> action)
+        public async static Task<T> ExecuteWithFileLocked<T>(string filePath, Func<bool, Task<T>> action)
```

Comment:
Yes, I'm going to do that in a PR that follows. I need to push this because it is blocking the CI